addons/website_sale_loyalty/data/loyalty_demo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="website" id="website.default_website">
+        <field name="has_loyalty">True</field>
+        <field name="loyalty_id" ref="loyalty.program" />
+    </record>
+
+    <record model="loyalty.reward" id="loyalty.reward_01">
+        <field name="image_1920" type="base64" file="website_sale_loyalty/static/img/gift.svg"/>
+    </record>
+
+    <record model="loyalty.reward" id="loyalty.reward_02">
+        <field name="image_1920" type="base64" file="website_sale_loyalty/static/img/discount.svg"/>
+    </record>
+
+    <record model="loyalty.reward" id="loyalty.reward_03">
+        <field name="image_1920" type="base64" file="website_sale_loyalty/static/img/discount.svg"/>
+    </record>
+
+    <record id="loyalty.simple_pen" model="product.product">
+        <field name="taxes_id" eval="False"/>
+        <field name="supplier_taxes_id" eval="False"/>
+    </record>
+
+    <record id="loyalty.discount_percent" model="product.product">
+        <field name="sale_ok" eval="False"/>
+        <field name="purchase_ok" eval="False"/>
+        <field name="taxes_id" eval="False"/>
+        <field name="supplier_taxes_id" eval="False"/>
+    </record>
+
+    <record id="loyalty.discount_fix" model="product.product">
+        <field name="sale_ok" eval="False"/>
+        <field name="purchase_ok" eval="False"/>
+        <field name="taxes_id" eval="False"/>
+        <field name="supplier_taxes_id" eval="False"/>
+    </record>
```

Comment:
![image](https://user-images.githubusercontent.com/30048408/118527620-c2916d80-b741-11eb-830d-81b23f794eb2.png)

Not too sure why I can't reply to this anymore but ok :+1: 
I thought we could have a new order in here so we don't land on an empty loyalty section in `/my` but it's not a big deal in the big scheme of things :smile: 
The less code the better after all